CalibTracker/SiStripChannelGain/test/7TeVData/Validation_Compute_cfg.py

```diff
@@ -2,9 +2,7 @@
 
 process = cms.Process("APVGAIN")
 
-process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi")
-process.load("Geometry.TrackerGeometryBuilder.trackerGeometry_cfi")
-process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
```

Comment:
Can you please move process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff") ?